package.json

```diff
@@ -55,7 +55,7 @@
 		"detect-indent": "6.1.0",
 		"detect-newline": "3.1.0",
 		"eslint-fix-utils": "^0.2.0",
-		"package-json-validator": "^0.10.0",
+		"package-json-validator": "~0.13.1",
```

Comment:
D'oh, sorry, I meant to comment on this - is the change from `^` to `~` intentional?